fdbrpc/Locality.cpp

```diff
@@ -185,6 +185,29 @@ ProcessClass::Fitness ProcessClass::machineClassFitness( ClusterRole role ) cons
 			default:
 				return ProcessClass::WorstFit;
 		}
+	case ProcessClass::RateKeeper:
+		switch( _class ) {
+			case ProcessClass::RateKeeperClass:
+				return ProcessClass::BestFit;
+			case ProcessClass::StatelessClass:
+				return ProcessClass::GoodFit;
+			case ProcessClass::MasterClass:
+				return ProcessClass::OkayFit;
+			case ProcessClass::ResolutionClass:
+				return ProcessClass::OkayFit;
+			case ProcessClass::TransactionClass:
+				return ProcessClass::OkayFit;
+			case ProcessClass::ProxyClass:
+				return ProcessClass::OkayFit;
```

Comment:
I'd vote for having MasterClass be OkayFit, as this is being split out of the master anyway, and making the rest be PoorFit. This way we prefer to put it somewhere else, but if there's no extra stateless processes, then we prefer to merge it back into the master, which is where it came from anyway.  (And I'd vote to do the same with DataDistribution as well.)  

I think if performance dashboard was running, you'd see a regression, because we don't run any extra stateless processes (I think), and thus you'll fall into the OkayFit.  When this happened with ClusterController, the extra latency caused by co-locating a proxy and cluster controller caused enough of a performance degradation that it was noticeable.